bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "url",
+  "main": "url.js",
+  "homepage": "https://github.com/Polymer/URL",
+  "authors": [
+    "https://github.com/Polymer/URL/graphs/contributors"
+  ],
+  "description": "URL parser in JavaScript",
+  "keywords": [
+    "url",
+    "urlsearchparams",
+    "whatwg",
+    "w3c"
+  ],
+  "license": "https://github.com/Polymer/URL#license",
```

Comment:
This one is not right since this code is public domain. Not same license as Polymer.